### DISCUSSION TITLE
Robin boundary example missing

### DISCUSSION THREAD

**Discussion by jobordu** [gismo/gismo#582] [Q&A]
Hi everybody, 

New user of G+Smo here :). 

I'm trying to modify the poisson_example.cpp to have Robin boundary without success.

The documentation is not clear on how to define the gsFunctionExpr for the Robin boundary.

Jonathan

**Comment by jobordu** [comment]
@filiatra @stefantakacs are Robin Boundaries supported by G+Smo ?

**Comment by jobordu** [comment]
Can we now apply robin boundaries ? @filiatra @stefantakacs

**Comment by hverhelst** [comment]
@jobordu is this still relevant for you?

**Reply by songbuling** [reply]
> @jobordu is this still relevant for you?
In fact, I had the same difficulty about setting the Robin boundary. 
And I wonder how to write it, especially the the gsFunctionExpr for the Robin boundary.I would be very grateful for your reply,thank you

**Comment by hverhelst** [comment]
See issue #755
